tapdb/universe_perf_test.go

```diff
@@ -180,19 +181,20 @@ func formatSize(bytes int64) string {
 
 // TestUniverseIndexPerformance tests query performance with indices.
 func TestUniverseIndexPerformance(t *testing.T) {
-	if testing.Short() {
-		t.Skip("skipping index performance test in short mode")
-	}
+    if testing.Short() {
```

Comment:
Incorrect formatting. Probably need to configure your IDE to use 8 spaces for tabs by default.